### PR TITLE
Perp 4075/analyze pool

### PR DESCRIPTION
**Pull request by lvn-rusty-dragon**

### CODE REVIEW

**Review by lvn-alduin — APPROVED** [review]
Looks good!

File: frontend/src/api/queries/Market.ts [hunk 1]
```diff
@@ -75,50 +80,88 @@ const marketFromResponse = (response: ResponseMarket): Market => {
...
   }
-  const outcomeTotalTokens = BigNumber(response.total_tokens);
+  const oddsForOutcome = totalProduct
+    .div(response.pool_tokens)
+    .div(totalOddsWeights)
+  let oddsWeightForOutcome = BigNumber(1)
+  for (const outcome of market.outcomes) {
+    oddsWeightForOutcome = oddsWeightForOutcome.times(outcome.pool_tokens)
+  }
+  oddsWeightForOutcome = oddsWeightForOutcome.div(response.pool_tokens)
```
Comment (lvn-alduin) [inline frontend/src/api/queries/Market.ts:127]: @lvn-rusty-dragon coming back to this I realized that `oddsWeightForOutcome` is not being used

Comment (snoyberg) [reply frontend/src/api/queries/Market.ts:127]: Doh, you're right, looks like I ended up implementing things twice. You mind cleaning that up? Sorry.

Comment (lvn-alduin) [reply frontend/src/api/queries/Market.ts:127]: So just not using `oddsWeightForOutcome` at all, right?

Comment (lvn-rusty-dragon) [reply frontend/src/api/queries/Market.ts:127]: Correct. Though probably hold off on this until the new math model is in place.

**Pull request merged**